fiscal_epos_print/static/src/js/pos_order_mgmt.js

```diff
@@ -0,0 +1,57 @@
+odoo.define("fiscal_epos_print.pos_order_mgmt", function (require) {
+    "use strict";
+
+    var core = require("web.core");
+    var pos_order_mgmt = require('pos_order_mgmt.widgets');
+    var epson_epos_print = require('fiscal_epos_print.epson_epos_print');
+    var _t = core._t;
+    var OrderListScreenWidget = pos_order_mgmt.OrderListScreenWidget;
+    var eposDriver = epson_epos_print.eposDriver;
+
+    OrderListScreenWidget.include({
+        _prepare_order_from_order_data: function (order_data, action) {
+            var order = this._super(order_data, action);
+            if (action === 'print') {
+                order.refund_report = order_data.refund_report;
+                order.refund_date = order_data.refund_date;
+                order.refund_doc_num = order_data.refund_doc_num;
+                order.refund_cash_fiscal_serial = order_data.refund_cash_fiscal_serial;
+            }
+            else if (action === 'return') {
+                order.refund_report = order_data.fiscal_z_rep_number;
+                order.refund_date = order_data.fiscal_receipt_date;
+                order.refund_doc_num = order_data.fiscal_receipt_number;
+                order.refund_cash_fiscal_serial = order_data.fiscal_printer_serial;
+            }
+            // for action === 'copy' we don't need to do anything
+
+            return order;
+        },
+
+        sendToFP90Printer: function(receipt, printer_options) {
+            for (var i = 0; i < printer_options.length; i++){
+                var printer_option = printer_options[i];
+                var fp90 = new eposDriver(printer_option, this);
+                fp90.printFiscalReceipt(receipt);
+            }
```

Comment:
Qui dovresti poter usare
```
            var fp90 = new eposDriver(printer_options, this);
            fp90.printFiscalReceipt(receipt);
```
senza for, come su v12

Reply:
Ok!